sys-libs/timezone-data/timezone-data-2021a-r1.ebuild

```diff
@@ -0,0 +1,196 @@
+# Copyright 1999-2022 Gentoo Authors
```

Comment:
You do not need to add `sys-libs/timezone-data` to portage-stable, because coreos-overlay already has that.

Reply:
Oh, indeed. Thanks, will drop it from the PR.